<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgGeneral</class>
 <widget class="QWidget" name="wdgGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wdgGeneral</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="cobxCurrentFont"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblFontName">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Font name:</string>
       </property>
       <property name="buddy">
        <cstring>ledFontName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ledFontName">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblColorTexture">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Color texture:</string>
       </property>
       <property name="buddy">
        <cstring>ledColorTexture</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="ledColorTexture">
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnColorTexture">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="folder-open"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblAlphaTexture">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Alpha texture:</string>
       </property>
       <property name="buddy">
        <cstring>ledAlphaTexture</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="ledAlphaTexture">
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAlphaTexture">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="folder-open"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblMaxHeigthOfChars">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Maximum height of characters:</string>
       </property>
       <property name="buddy">
        <cstring>sbxMaxHeigthOfChars</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="sbxMaxHeigthOfChars">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblDistanceBetweenChars">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Distance between characters:</string>
       </property>
       <property name="buddy">
        <cstring>sbxDistanceBetweenChars</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="sbxDistanceBetweenChars">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
